Tests/IntegrationTest.php

```diff
@@ -329,7 +329,12 @@ public function testRunCommandWithDefaultPurger(): void
         $kernel->boot();
         $container = $kernel->getContainer();
 
-        $em       = $this->createConfiguredMock(EntityManagerInterface::class, ['getConnection' => $this->createMock(Connection::class), 'getEventManager' => $this->createMock(EventManager::class)]);
+        $em       = $this->getMockBuilder(EntityManagerInterface::class)
+            ->addMethods(['wrapInTransaction'])
```

Comment:
Argh! Looks like this method does not exist on low versions of PHPUnit

Reply:
See https://github.com/sebastianbergmann/phpunit/commit/d618fa3fda437421264dcfa1413a474f306f79c4

Reply:
I guess you will need to resort to `setMethods` :see_no_evil: or maybe we drop support for PHP 7.1. @derrabus , WDYT? This is a leaf package, so it might make sense to do so?

Reply:
PHPUnit 7 is no longer supported (https://phpunit.de/supported-versions.html) but we need PHPUnit 7 to support PHP 7.1... I agree with both of your solutions (`setMethods` or drop support of PHP 7.1)

Reply:
Third solution: tag the test with `@requires PHPUnit 8.3.0`

Reply:
This 3rd solution is a good compromise. Am I using it or still waiting?

Reply:
You can try using that last workaround, yeah :)

Reply:
@greg0ire Of all 3.4.x installations, 0.8% use PHP 7.1.

https://packagist.org/packages/doctrine/doctrine-fixtures-bundle/php-stats#3.4

I think we can kill PHP 7.1. And looking at composer.json, there's some more, we could throw out, like Symfony 3 or DoctrineBundle 1.